lib/takeaway.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'menu'
+require_relative 'keys'
+require 'twilio-ruby'
+
+class Takeaway
+  attr_reader :menu, :order, :order_total
+
```

Comment:
Good that you injected the Menu class into the Takeaway class instead of making it a hard coded dependency, makes isolating your tests with a double easier.